lib/utils/parseJson.js

```diff
@@ -0,0 +1,11 @@
+const { parse } = require('jju/lib/parse');
```

Comment:
Only `utils.js` relies on `fs`. `parse` module should be working fine in isolation, getting rid of `fs` dependency, comparing when importing the module entirely:

```js
const jju = require('jju')
```